content/events/2024-04-25-ESG-annual-meeting/index.md

```diff
@@ -0,0 +1,12 @@
+---
+site: Berlin
+tags: meeting, General Assembly
+title: "EuroScienceGateway General Assembly Meeting 2024"
+starts: 2024-10-24
+ends: 2024-10-24
+organiser: ALU Freiburg Team
+location:  ParkInn Hotel, Alexanderplatz 7, 10178, Berlin
+supporters: EU
+---
+
+The annual meeting of the European project [EuroScienceGateway](https://eurosciencegateway.eu), where representatives of each Consortium partner meet to make strategic decisions. 
```

Comment:
This sentence can be replaces by:

The annual meeting of the Horizon Europe project EOSC [EuroScienceGateway](https://eurosciencegateway.eu) takes place in Berlin after the [EOSC Symposium](https://eosc.eu/symposium2024). Representatives of each consortium partner meet to join the general assembly and make strategic decisions for the remaining project time.

Contact: Anika Erxleben-Eggenhofer (erxleben@informatik.uni-freiburg.de)